libs/clients/criminal-record/src/lib/criminalRecordApi.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common'
+import { CriminalRecord } from './criminalRecordApi.types'
+import { CrimeCertificateApi } from '../../gen/fetch'
+
+@Injectable()
+export class CriminalRecordApi {
+  constructor(private readonly api: CrimeCertificateApi) {}
+
+  public async getCriminalRecord(ssn: string): Promise<CriminalRecord> {
+    const contentBlob = await this.api.apiPdfCreatePersonIdGet({
+      personId: ssn,
+    })
+    const contentArrayBuffer = await contentBlob.arrayBuffer()
+    const contentBase64 = Buffer.from(contentArrayBuffer).toString('base64')
```

Comment:
Another solution would be to upload it to the download S3 bucket... but might be better to look at the download service?